packages/thirdweb/src/extensions/erc721/lazyMinting/write/createDelayedRevealBatch.test.ts

```diff
@@ -51,7 +51,7 @@ const realNFTs = [
   },
 ];
 
-describe("createDelayedRevealedBatch", () => {
+describe.skip("createDelayedRevealedBatch", () => {
```

Comment:
I will make a new test - today- and it will be better

Reply:
make it as part of this PR we can't not have a valid test for this before we ship it

Reply:
test added!